### DISCUSSION TITLE
Add a connection indicator

### DISCUSSION THREAD

**Discussion by Karatzie** [fl4p/batmon-ha#140] [Ideas]
Hi Fabian.

I would appreciate a connection indicator for the bluetooth devices.
Many users use batmon with multiple batteries.
If batmon increments a counter every time data is received from a bluetooth device, then we have a value that changes with every transmission.
Each time this counter changes, a symbol could be briefly displayed for eachb device in an overview.

BR
Andreas

**Comment by fl4p** [comment]
Hi Andres,
I'll add a sample counter with the next release

**Comment by fl4p** [comment]
Also note that batmon uses HA's expiry api so values expire if the connection to the bms is interrupted for some time

**Comment by fl4p** [comment]
This has been added with https://github.com/fl4p/batmon-ha/commit/1ac89fc2c14c205cc66495943b316f1f319021ac

might be relevant if you use esphome: i creeated a feature request for HA's availability api https://github.com/esphome/feature-requests/issues/2438

**Comment by Karatzie** [comment]
Thanks a lot.
The sample counter works and I have a small indicator for each data excange.

BR
Andreas
